lib/elixir/lib/module.ex

```diff
@@ -543,6 +553,103 @@ defmodule Module do
   @callback __info__(:md5) :: binary()
   @callback __info__(:module) :: module()
 
+  @doc """
+  Returns information about module attributes used by Elixir.
+
+  See the "Module attributes" section in the module documentation for more
+  information on each attribute.
+
+  ## Examples
+
+      iex> map = Module.reserved_attributes()
+      iex> Map.has_key?(map, :moduledoc)
+      true
+      iex> Map.has_key?(map, :doc)
+      true
+
+  """
+  @doc since: "1.13.0"
```

Comment:
@josevalim in case you want to backport it to v1.12 :)

```suggestion
  @doc since: "1.12.0"
```